.github/workflows/deploy-frontend.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy Frontend
+on: workflow_dispatch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Move Files
+      run: mv frontend/html .; mv frontend/config .; mv frontend/sources .
+    - name: Create env-config.ini
+      run: touch config/env-config.ini; echo "env=prod">config/env-config.ini
+    - name: Clean ec2 instance deployment
+      uses: appleboy/ssh-action@master
+      with:
+        host: 52.14.239.79
+        username: ec2-user
+        key : ${{ secrets.EC2_KEY }}
+        port: 22
+        script: rm -r /var/www/html/*; rm -r /var/www/sources/*; rm -r /var/www/config/*
```

Comment:
This line will fail on the first call because `rm -r` fails if there is no file/directory present. Change the `rm`'s to `rm -rf` to continue on error.

Reply:
Ah, yeah. That would also be pretty bad if a deploy fails and then the next one gets stuck because there's nothing to delete. I'll adjust that real quick.